manifests/dotnet.yml

```
@@ -552,6 +552,13 @@ tests/:
   stats/:
     test_miscs.py:
       Test_Miscs: missing_feature
+  test_baggage.py:
+    Test_Baggage_Headers_Basic: v3.6.0
+    Test_Baggage_Headers_Malformed: v3.6.0
+    Test_Baggage_Headers_Malformed2: v3.6.0
```

```suggestion
    Test_Baggage_Headers_Malformed_2: v3.6.0
```

Can we make this change for readability, similarly to how we do it for Scenarios? (e.g. `TRACING_CONFIG_NONDEFAULT_2`

---

I can make this change in a new PR! I just wanna merge this PR ASAP and the CI is taking extraaa long to run. I'll make sure to introduce this change in a separate PR.